config/interstitial.go

```diff
@@ -0,0 +1,267 @@
+package config
+
+// This is a first pass at providing a priority list for building out interstitial sizes.
+// It is quite possible that different strategies may develop, requiring a configurable
+// list to satisfy the needs of different hosting companies. So we start with a hardcoded
+// list in the config module, to make it more straightforward to move to an actual
+// configurable solution once the need develops and the use cases can be defined.
+
+type InterstitialSize struct {
+	Width  uint64
+	Height uint64
+}
+
+type InterstitialSizes []InterstitialSize
+
+var ResolvedInterstitialSizes = InterstitialSizes{
```

Comment:
where was the size list sourced from? Might be worth adding a comment.

Reply:
It was sourced from delivery statistics from Appnexus. A list of sizes sorted by size (larger first) and frequency(more common sizes first) with what seemed to be a good weighting factor balancing the two factors.